blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -0,0 +1,347 @@
+import React, {
+  createRef, useCallback, useEffect, useReducer, useState,
+} from 'react';
+
+import ArticleDate from '@wpmedia/date-block';
+import styled from 'styled-components';
+import getThemeStyle from 'fusion:themes';
+import getTranslatedPhrases from 'fusion:intl';
+import { useContent } from 'fusion:content';
+
+import { Image, isServerSide } from '@wpmedia/engine-theme-sdk';
+import { extractResizedParams, extractImageFromStory } from '@wpmedia/resizer-image-block';
+import {
+  Byline,
+  Heading,
+  SecondaryFont,
+} from '@wpmedia/shared-styles';
+import {
+  reduceResultList,
+  resolveDefaultPromoElements,
+} from './helpers';
+
+const ReadMoreButton = styled.button`
+  background-color: ${(props) => props.primaryColor};
+  font-family: ${(props) => props.primaryFont};
+
+  &:not(:disabled):not(.disabled):active:hover,
+  &:not(:disabled):not(.disabled):hover:hover {
+    background-color: ${(props) => props.primaryColor};
+  }
+`;
+
+const ResultsItem = React.memo(React.forwardRef(({
```

Comment:
Pulling this out to it's own file would make this file a bit more readable, and possibly allow us to in the future utilize this component in other list blocks

Reply:
I was hoping you didn't notice that.  I definitely thought about this... I suppose since this story is carried on, I will undertake that and update/make the appropriate tests for it.

Reply:
so this is resolved, right?

Reply:
no, it doesn't seem like it is

Reply:
I can if you would like.  It's going to involve pulling some tests and this into their own files... maybe an hour?

Reply:
done and committed